test/Altinn.AccessManagement.Tests/Controllers/V2RightsInternalControllerTest.cs

```diff
@@ -127,8 +127,7 @@ public static TheoryData<SeedGetRightsDelegationsOffered> Seeds() => new()
 
                 WithAssertDbDelegationsNotEmpty,
                 WithAssertDbLastDelegationToUserIsRevoked(OrganizationSeeds.VossConsulting.Defaults, PersonSeeds.Paula.Defaults),
-                WithAssertResponseStatusCodeSuccessful,
-                WithAssertEmptyDelegationList),
```

Comment:
Guess this change might be the reason for the failing integration test?

Reply:
fixed after merging with main